### DISCUSSION TITLE
没有找到 im-select

### DISCUSSION THREAD

**Discussion by lizhuoran1019** [nshen/learn-neovim-lua#131] [Q&A]
win10 + windows terminal + wsl2 + ubuntu20.04 + neovim0.8
下载了im-select.exe 放在 ~/.config/nvim 中，但是还显示没有找到im-select
求大佬们帮帮忙
![image](https://user-images.githubusercontent.com/43104914/196584510-212d0fa1-3f78-4358-b8a0-92c274d84509.png)
![image](https://user-images.githubusercontent.com/43104914/196584648-256030e5-c693-454a-9da2-ba0761e82924.png)

**Comment by nshen** [comment]
可能是代码有问题，我用的 `:lua vim.fn.executable('im-select')` 应该是判断全局的

把 im-select.lua 中两段判断都删掉试试？

```
if vim.fn.executable("im-select") ~= 1 then
    vim.notify("没有找到 im-select 无法切换输入法, https://github.com/daipeihust/im-select")
    return
  end
```

**Reply by lizhuoran1019** [reply]
删除之后不报错，但是im-select不工作。
在insert模式使用中文输入后，返回normal模式不会自动切换为英文输入法

**Reply by lizhuoran1019** [reply]
![image](https://user-images.githubusercontent.com/43104914/196983066-5491d26b-2d3e-4b99-8ccb-82d953d4126f.png)
我把im-select.lua中的代码改成这样就可以了。
而且我意识到，只有使用esc返回normal模式时im-select才工作，使用\<C-c\>无法触发im-select，请问这个有办法解决吗？

**Reply by nshen** [reply]
我是用esc返回normal模式的，<C-c> 是什么？

**Reply by lizhuoran1019** [reply]
ctrl+c

**Reply by nshen** [reply]
没明白 ctrl+c  为啥要触发im-select
